hack/verify-codecgen.sh

```diff
@@ -86,12 +86,16 @@ index=(${result})
 # Build codecgen from Godeps.
 # However, we need to install godep first.
 # We make some tricks with GOPATH variable to make it work with Travis.
-_gopath=${GOPATH}
-export GOPATH="${_tmpdir}"
-go get -u github.com/tools/godep 2>/dev/null
-go install github.com/tools/godep 2>/dev/null
-GODEP="${_tmpdir}/bin/godep"
-export GOPATH=${_gopath}
+if [[ -n "${TRAVIS:-}" ]]; then
```

Comment:
yeah, I'd vote to just remove installing godep entirely. we can make the script bail out of godep isn't installed.